db/migrate/20230904121544_create_journal_entries.rb

```diff
@@ -0,0 +1,17 @@
+class CreateJournalEntries < ActiveRecord::Migration[7.0]
+  def up
+    create_table :journal_entries do |t|
+      t.integer "journal_id", :null => false, index: true, foreign_key: true
+      t.integer "user_id"
```

Comment:
why is the user_id here? I do not see any association with the user model in this PR